src/providers/toast.service.ts

```diff
@@ -14,4 +22,20 @@ export class ToastService {
     this.faveToast = this.toast.create({message: txt, position: 'bottom', showCloseButton: true});
     this.faveToast.present();
   }
+  noOriginOrDestinationToast(): void{
+    if (this.originDestination) {
+      this.originDestination.dismiss();
+    }
+    let txt = 'You must select an origin and destination from the autocomplete dropdowns above in order to search the schedule';
+    this.originDestination = this.toast.create({message: txt, position: 'bottom', showCloseButton: true});
+    this.originDestination.present();
+  }
+  noLocationToast(): void{
```

Comment:
`noLocationToast` and `noOriginOrDestinationToast` were added by me. Maybe they should just use the ToastController on that one page. Idk

Reply:
What I was trying to accomplish was making a toast that dismisses and presents itself. And I went down a rabbit hole. Anyway.